stores/metadata.go

```diff
@@ -1507,6 +1507,10 @@ func (s *SQLStore) RenameObjects(ctx context.Context, bucket, prefixOld, prefixN
 				gorm.Expr(sqlConcat(tx, "?", "SUBSTR(object_id, ?)")), prefixNew,
 				utf8.RuneCountInString(prefixOld)+1, prefixOld+"%",
 				utf8.RuneCountInString(prefixOld), prefixOld, sqlWhereBucket("objects", bucket))
+
+			if !isSQLite(tx) {
+				inner = tx.Raw("SELECT * FROM (?) as i", inner)
```

Comment:
What is the subquery for?

Reply:
The subquery is necessary on MySQL. This is actually an actual bug in `renterd` on MySQL, if you force rename objects on a MySQL instance it should throw an error.